internal/parser/parser.go

```diff
@@ -1036,8 +1036,8 @@ func (p *parser) parseObjectLiteral() ast.Expression {
 }
 
 func (p *parser) parseParenExpression() ast.Expression {
-	pos, _ := p.open(token.LPAREN, token.RPAREN)
```

Comment:
Why the name change?

Reply:
cause I forgot to revert it back, fixing now